Newsletters: add 419 (2026-08-21) (#2850)
bitcoinops/bitcoinops.github.io: _posts/en/newsletters/2026-08-21-newsletter.md

  to Delving Bitcoin the [responsible disclosure][topic responsible disclosures]
  of a vulnerability that affected LND versions before [0.20.0][lnd v20.0],
  which fixed it in February 2026. Operators running an older version should
  upgrade. To Teinturier's knowledge, no one was affected.

  Before that version, an LND node would forget about a collaboratively closed
  channel immediately after the closing transaction's first onchain
  confirmation, losing its protection against chain reorgs. If a reorg then
  removed that confirmation, an attacker could publish an old, revoked
  commitment transaction for the channel and because the node had already
  forgotten the channel, it would not respond with a penalty transaction,
  letting the attacker drain all of the channel's funds.

  The vulnerability was discovered in February 2025 and fixed a year later in
  [LND #10331][] (see [Newsletter #389][news389 lnd10331]), which makes a node
  wait for more confirmations before considering a channel close final (at
  least six, following [BOLT5][]'s reorg-safety handling). Teinturier's post
  includes a regtest reproduction and a timeline of the disclosure.

> - **BIP draft for `rawtr()` output script descriptor**: Jean Pablo [posted][rawtr ml]

Author: ```suggestion
- **Draft BIP for `rawtr()` output script descriptor**: Jean Pablo [posted][rawtr ml]
```

I think this follows more what we have done in the past for draft BIP news items

  to the Bitcoin-Dev mailing list about a BIP proposal for the `rawtr()`
  [output script descriptor][topic descriptors]. This descriptor has been available in
  Bitcoin Core since version 24.0, but it was never specified in a proper BIP. Several
  implementations route around the problem by either not supporting it,
  or quoting other BIPs. The proposal aims to close this gap. The BIP draft is available,
  together with test vectors, and it is being discussed under [BIPs #2251][].

>   A `rawtr()` descriptor can be used to express a P2TR output directly by its output key,

Author: Perhaps mentioning no tweak as well.

Author: Perhaps we also move this earlier in the writeup? (what is it, then why it needs a BIP?)

  upgrade. To Teinturier's knowledge, no one was affected.

  Before that version, an LND node would forget about a collaboratively closed
  channel immediately after the closing transaction's first onchain
  confirmation, losing its protection against chain reorgs. If a reorg then
  removed that confirmation, an attacker could publish an old, revoked
  commitment transaction for the channel and because the node had already
  forgotten the channel, it would not respond with a penalty transaction,
  letting the attacker drain all of the channel's funds.

  The vulnerability was discovered in February 2025 and fixed a year later in
  [LND #10331][] (see [Newsletter #389][news389 lnd10331]), which makes a node
  wait for more confirmations before considering a channel close final (at
  least six, following [BOLT5][]'s reorg-safety handling). Teinturier's post
  includes a regtest reproduction and a timeline of the disclosure.

- **BIP draft for `rawtr()` output script descriptor**: Jean Pablo [posted][rawtr ml]
  to the Bitcoin-Dev mailing list about a BIP proposal for the `rawtr()`
  [output script descriptor][topic descriptors]. This descriptor has been available in

>   Bitcoin Core since version 24.0, but it was never specified in a proper BIP. Several

Reviewer: ```suggestion
  Bitcoin Core since version 24.0, but had not yet been specified in a BIP. Several
```

  implementations route around the problem by either not supporting it,
  or quoting other BIPs. The proposal aims to close this gap. The BIP draft is available,

>   together with test vectors, and it is being discussed under [BIPs #2251][].

Reviewer: Stumbled on the “it”:
```suggestion
  or quoting other BIPs. The proposal aims to close this gap. The BIP draft and test vectors are available 
  and being discussed under [BIPs #2251][].
```
